src/jit/lsra.cpp

```diff
@@ -4148,6 +4161,14 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
             useCandidates  = allRegs(registerType);
         }
 
+#ifdef ARM_SOFTFP
```

Comment:
I see. There should at least be a comment about the assumption that the arg registers are in sequential order.